JSQMessagesViewController/Views/JSQMessagesComposerTextView.m

```diff
@@ -163,6 +164,11 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
 - (void)paste:(id)sender
 {
     if (!self.pasteDelegate || [self.pasteDelegate composerTextView:self shouldPasteWithSender:sender]) {
+        if (!shouldPasteRTF) {
+            [self setText:[UIPasteboard generalPasteboard].string];
+            [self.pasteDelegate composerTextView:self didPasteWithSender:sender];
```

Comment:
its possible to get here with a nil pasteDelegate, is that ok?

Reply:
Yes. In that case it uses default methods to paste.